jfrog-cli/artifactory/cli.go

```diff
@@ -1126,6 +1126,15 @@ func getBuildScanFlags() []cli.Flag {
 	}...)
 }
 
+func getBuildAddGitFlags() []cli.Flag {
+	return []cli.Flag{
+		cli.StringFlag{
+			Name:  "config",
+			Usage: "[Optional] Path to configuration file.` `",
```

Comment:
Path to configuration file. --> Path to **a** configuration file.

Reply:
Fixed.